app/src/main/java/com/google/samples/apps/sunflower/GardenActivity.kt

```diff
@@ -32,6 +33,10 @@ class GardenActivity : AppCompatActivity() {
         // Displaying edge-to-edge
         WindowCompat.setDecorFitsSystemWindows(window, false)
 
-        setContentView<ActivityGardenBinding>(this, R.layout.activity_garden)
+        setContent {
```

Comment:
Unfortunately this has to be `AppCompatActivity` since `HomeScreen` is using `setSupportActionBar`. This relates to #829. I'll add a comment in the code though.